sdk/nodejs/cmd/run/run.ts

```diff
@@ -171,7 +171,7 @@ export function run(argv: minimist.ParsedArgs): void {
         }
         else {
             log.error(`Running program '${program}' failed with an unhandled exception:`);
-            log.error(util.format(err));
+            log.error(err.stack || err.message);
```

Comment:
`util.format` is bad at printing out gRPC exceptions. It doesn't believe that gRPC exceptions are `Error`s and prints out the whole exception as if it were an object. Printing out the stack (or just the message, if no stack is available) is a lot better in that it only prints the useful information - the stack and the exception message.